sdk/src/SafeConvert.hpp

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
Looks like this has been added again, when it already exists at a different location. We should figure out what's causing this to be imported in the wrong place and fix if possible. Otherwise, just delete this file.

Reply:
Yes, it's referenced here: https://github.com/rstudio/rstudio-launcher-plugin-sdk/blob/a81626424f7e86961748c232bc9f9c97f5e5558b/tools/pull-shared-core.sh#L345

Reply:
Okay, I am finding that removing this causes other issues with compilation. ^